test-requirements.txt

```diff
@@ -1,3 +1,4 @@
+-r requirements.txt
```

Comment:
I find this as an anti-pattern. Test requirements should not also include requirments.

While I find the refactoring of tox.ini very useful, I am afraid that the scope of the refactoring keeps growing and instead we should make smaller incremental improvements on it.

Reply:
I don't think that it's an anti-pattern in general. It depends on the use-case. In this case, it's probably fine for now since I'm going to fix dependency management along with packaging anyway...